src/gdswzd_mod.f90

```diff
@@ -1,40 +1,30 @@
+!> @file
+!! @brief Driver module for gdswzd routines.
+!!
+!! @date July 2021
+!! @author George Gayno, Kyle Gerheiser
```

Comment:
You should not add your name unless you also developed this code. ALso the date should be the date the file was first added, not the current date.